addon/services/apollo.js

```diff
@@ -86,10 +88,18 @@ export default Service.extend({
    * @return {!Object}
    * @public
    */
-  clientOptions: computed(function() {
+   clientOptions: computed(function() {
     const apiURL = this.get('apiURL');
+    const credentials = this.get('credentials');
     const middlewares = this.get('middlewares');
-    const networkInterface = createNetworkInterface({ uri: apiURL });
+    const networkInterfaceOptions = {
+      uri: apiURL,
+      opts: {}
+    }
+    if (isPresent(credentials) {
```

Comment:
Missing close parentheses here is causing a test failure.

Reply:
fixed